env.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"runtime"
+
+	"github.com/la5nta/pat/internal/buildinfo"
+)
+
+func envHandle(_ []string) {
+	writeEnvAll(os.Stdout)
```

Comment:
Looks useful enough for now. Do you want to extend it later to take an argument similar to `go env`?

Reply:
No immediate plans, but maybe? I have been missing a way to programmatically get various directory paths, so. that's the main motivation for this.